docs/app/pages/components/components-sections/timeline/timeline/timeline.component.ts

```diff
@@ -0,0 +1,84 @@
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+
+@Component({
+    selector: 'uxd-components-timeline',
+    templateUrl: './timeline.component.html'
+})
+@DocumentationSectionComponent('ComponentsTimelineComponent')
+export class ComponentsTimelineComponent extends BaseDocumentationSection {
+
+    constructor() {
+        super(require.context('./snippets/', false, /\.(html|css|js|ts)$/));
+    }
+
+    _developer = chance.name();
```

Comment:
Variable declarations should come before the constructor. Also, if this is a private variable add the 'private' keyword.

Reply:
Also, does developer need to be an instance variable? Could the chance just not be used inline, eg:

```typescript
this.tickets.unshift({
    color: 'alternate2',
    shortDate: month + ' ' + day + suffix,
    title: 'Addition of Timeline component',
    date: now,
    url: '#',
    id: '1234',
    status: 'updated by',
    assignee: chance.name()
});
```

Reply:
chance.name() now called inline. Unneeded _developer variable removed.